examples/ignition/bootkube-controller.yaml

```diff
@@ -133,7 +133,7 @@ storage:
           # Wrapper for bootkube start
           set -e
           BOOTKUBE_ACI="${BOOTKUBE_ACI:-quay.io/coreos/bootkube}"
-          BOOTKUBE_VERSION="${BOOTKUBE_VERSION:-v0.2.6}"
+          BOOTKUBE_VERSION="${BOOTKUBE_VERSION:-v0.3.0}"
```

Comment:
Maybe minor, but wanted to note that you're using two different versions of bootkube (based on the referenced hash in the docs). Assuming this is fine because the docs reference a commit that is not in a release yet, but needed for the `render` step -- whereas the container version here is only used for the `start` step -- which is unchanged. In this case seems fine, but in future should try and sync these up to avoid hard to debug / unknown states.